frontend/frontend-auto-translate.php

```diff
@@ -334,7 +334,10 @@ protected function translate_terms_list( $query_var, $taxonomy ) {
 				// We got an unexpected query var, let return it unchanged.
 				return $query_var;
 			}
-			$slug = $this->get_translated_term_by( 'slug', $slug, $taxonomy );
+			$found_slug = $this->get_translated_term_by( 'slug', $slug, $taxonomy );
+			if ( is_string( $found_slug ) ) {
+				$slug = $found_slug;
+			}
```

Comment:
This adds dead code so I would keep as is for now. The issue is coming from the doc of `get_translated_term_by()` as here, we can't get anything than as a string.

Reply:
`get_translated_term_by()` *can* return something else [here](https://github.com/polylang/polylang/blob/dc8e447f57c1e2619882b8333c39dfb148b03f0f/frontend/frontend-auto-translate.php#L300)? Am I right?